src/DynamixelMotor.cpp

```diff
@@ -121,3 +127,39 @@ uint16_t DynamixelMotor::currentPosition()
 	return currentPosition;
 }
 
+DynamixelStatus DynamixelMotor::getCurrentPosition(uint16_t &pos)
+{
+	return read(DYN_ADDRESS_CURRENT_POSITION, pos);
+}
+
+DynamixelStatus DynamixelMotor::setComplianceMargins(byte cw_margin, byte ccw_margin, byte cw_slope, byte ccw_slope)
+{
+	DynamixelStatus status;
```

Comment:
I know this is currently not always used in the other functions, but the read/write also save the status in a mStatus variable, so it is best to use it.
I am fixing this in the other functions as well.